.travis.yml

```diff
@@ -22,6 +22,9 @@ script:
 
 ## lint the github PR title.
 - npm run travis-pr-title-lint
+## In case Travis is building a release tag, make sure we have a production build
+## before publishing to npm
+- NODE_ENV=production npm run build
```

Comment:
:+1:

I guess that we can use something like:
```
before_deploy:
- NODE_ENV=production npm run build
```
to run it only when we are release a new version on a tag build (https://docs.travis-ci.com/user/deployment/npm/#Running-commands-before-and-after-deploy)

Reply:
oh nice! I didn't know about that hook.

Reply:
That will keep the build running fast when we aren't doing a deploy